nntrainer/models/neuralnet.cpp

```diff
@@ -1191,4 +1194,24 @@ void NeuralNetwork::forEachLayer(
     fn(*ln, std::forward<RunLayerContext &>(ln->getRunContext()), user_data);
   };
 }
+
+void NeuralNetwork::exports(nntrainer::ExportMethods method,
+                            const std::string file_path) {
+  switch (method) {
+  case nntrainer::ExportMethods::METHOD_TFLITE: {
+    nntrainer::TfliteInterpreter interpreter;
+
+    /// We will call "serialize" method for the model which is already trained
+    /// or allocated. So, we need to call deallocateTensors first to make sure
+    /// `dealloc_weights == false`
```

Comment:
This comments are from https://github.com/nnstreamer/nntrainer/pull/1892#discussion_r868712315.